### PR TITLE
Application fixes

### PR DESCRIPTION
**Pull request by Rafal-1999**
I did all the fixes on the Slack list 😎  Without responsive at this moment. These changes improve the overall appearance of the application. Please check it and I am waiting for feedback. I will then upload the changes.

### CODE REVIEW

**Comment by adim2222** [comment]
This is actually great progress. Good job!

**Comment by Rafal-1999** [comment]
> This is actually great progress. Good job!

Thanks. So can I merge it?

File: src/common/MovieCard/styled.js [hunk 1]
```diff
@@ -16,6 +17,11 @@ export const CardContainer = styled.div`
     grid-template-columns: 1fr 1fr;
     grid-gap: 16px;
   }
+
+  &:hover {
```
Comment (DawidGaski) [inline src/common/MovieCard/styled.js:21]: Nice, you added the hover effect,

File: src/useYearFromDate.js
Comment (DawidGaski) [inline src/useYearFromDate.js:1]: looks ok, it works

File: src/useAddComma.js
Comment (DawidGaski) [inline src/useAddComma.js:1]: good, you remembered about that replace

File: src/common/Navigation/styled.js [hunk 2]
```diff
@@ -151,4 +154,9 @@ export const LinkNavigation = styled(NavLink)`
     &.active {
         border: 1px solid ${({ theme }) => theme.color.white};
     }
+    &:hover,
+    &:focus {
+        color: ${({ theme }) => theme.color.woodsmoke};
```
Comment (DawidGaski) [inline src/common/Navigation/styled.js:159]: gj

File: src/common/PersonCard/styled.js [hunk 3]
```diff
@@ -42,10 +42,16 @@ export const Card = styled.div`
     padding: 8px;
     min-width: 208px;
     min-height: 339px;
+    transition: all 0.2s linear;
 
     @media (min-width: 768px) {
         padding: 16px 16px;
     }
+
+    &:hover {
```
Comment (DawidGaski) [inline src/common/PersonCard/styled.js:51]: and again, well done

File: src/common/Navigation/index.js [hunk 4]
```diff
@@ -1,17 +1,30 @@
...
+    } else {
+        content = "people";
+    }
+
+    return (
+        <NavigationBar>
+            <Box>
+                <MainHeader><VideoIcon />Movies Browser</MainHeader>
+                <BoxNav>
+                    <LinkNavigation to={toMovieCard()}>MOVIES</LinkNavigation>
```
Comment (DawidGaski) [inline src/common/Navigation/index.js]: ` toMovieCard ` should names ` toPopularMovies` for better clarity

File: src/Routes/router.js [hunk 5]
```diff
@@ -4,6 +4,7 @@ import MoviesList from "./MoviesList";
 import SearchResults from "./SearchResults";
 import MoviePage from "./MoviePage";
 
+export const toPopularPeople = () => "/people";
 export const toMovieCard = () => "/movies";
```
Comment (DawidGaski) [inline src/Routes/router.js]: toMovieCard should names  toPopularMovies for better clarity

File: src/common/Navigation/index.js
Comment (DawidGaski) [inline src/common/Navigation/index.js:1]: Nice Work. components Navigation looks now better and performs more actions.

**Review by DawidGaski — APPROVED** [review]
approve ;)

File: src/useYearFromDate.js
Comment (Rafal-1999) [reply src/useYearFromDate.js:1]: Of course. I checked it.

File: src/useAddComma.js
Comment (Rafal-1999) [reply src/useAddComma.js:1]: Yes, I remembered about comma.

File: src/common/Navigation/index.js (hunk 4 above)
Comment (Rafal-1999) [reply src/common/Navigation/index.js]: It is already here 😄

**Pull request merged**